api/product/product.go

```
@@ -0,0 +1,7 @@
+package productHandler
+
+import "products/internal/db/product"
+
+type ProductHandler struct {
+	queries product.Querier
```

golangci-lint (2.11.4)

[error] field queries is unused

(unused)